FBShimmering/FBShimmeringLayer.m

```diff
@@ -118,7 +128,7 @@ - (void)layoutSublayers
 
 @end
 
-@interface FBShimmeringLayer ()
+@interface FBShimmeringLayer () <FBShimmeringCALayerDelegate, FBShimmeringCAAnimationDelegate>
```

Comment:
Instead of the macro, you could inline right here:

```objc
@interface FBShimmeringLayer ()
#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 100000
<CALayerDelegate, CAAnimationDelegate>
#endif
```